docs/CHANGELOG.asciidoc

```diff
@@ -18,6 +19,532 @@ This section summarizes the changes in each release.
+* Decreases the size of tray toggle {pull}25470[#25470]
+* Fixes the hours label container size {pull}31838[#31838]
+* Removes ID attribute {pull}30736[#30736]
+* Improves the plugin pre-build {pull}25267[#25267]
+Dashboard::
+* Removes `dashboardContext` function and makes Timelion, Vega, and Time Series 
+Visual Builder use `buildEsQuery` {pull}23227[#23227]
+* Shows the filter bar in dashboard embed mode {pull}32222[#32222]
+Design::
+* Updates dashboard design {pull}29896[#29896]
+* Adjusts side navigation interaction {pull}29978[#29978]
+* Converts Security UI from LESS to Sass {pull}25079[#25079]
+* Adds boilerplate Sass for Kibana core {pull}21185[#21185]
+* Makes dark mode overrides global {pull}31628[#31628]
+Discover::
+* Adds debug code to flaky field_data test {pull}15535[#15535]
+* Gets even more debug info for flaky field_data test {pull}17627[#17627]
+* Allows time ranges before the epoch in Discover {pull}31807[#31807]
+* Fixes Kibana Query Language handling of carriage return {pull}32150[#32150]
+* Improves warning messages when using Lucene-like syntax {pull}31837[#31837]
+* Update capitalization of KQL full name {pull}31631[#31631]
+* Adds placeholder text indicating the filters aggregation takes only Lucene and Query DSL {pull}31296[#31296]
+* Prevents sending request if it already failed during query parsing {pull}31219[#31219]
+* Fixes apps to always migrate legacy queries on state change {pull}33276[#33276]
+Geo::
+* Fixes panel cut off in K7 design {pull}29209[#29209]
+* Removes left-over geohash grid code {pull}30312[#30312]
+* Uses geo-tile aggregation instead of geohash precision {pull}29776[#29776]
+* Resolves URL dynamically when requesting EMS data {pull}25685[#25685]
+* Fixes EMS hotlink {pull}26868[#26868]
+* Fixes feature/align map config settings {pull}19450[#19450]
+* Removes `geotile_grid` aggregation from non-geospatial visualizations {pull}31354[#31354]
+* Coerces string doc-values to numbers if used in styling {pull}33657[#33657]
+Infrastructure::
+* Fixes server-side retrieval of the {kib} version {pull}31440[#31440]
+* Fixes react router import {pull}33659[#33659]
+* Adapts settings ui to saved object version type change {pull}30082[#30082]
+* Fixes potential color bugs {pull}26292[#26292]
+* Fixes auto refresh button on node detail page {pull}26426[#26426]
+* Changes the time range from the last hour to the last 5 minutes for the Waffle Map {pull}26278[#26278]
+* Passes flag in request to force BWC hit count {pull}26517[#26517]
+* Replaces redux source slice with constate container {pull}26121[#26121]
+* Changes node detail link to set time range to 1 hour {pull}26977[#26977]
+* Stops showing sidenav while loading. {pull}27119[#27119]
+* Fixes styling after breaking EUI changes {pull}27021[#27021]
+* Fixes graphql type generation after package upgrades {pull}26991[#26991]
+* Removes usage of `ts-optchain` in the browser {pull}27148[#27148]
+Kibana App::
```

Comment:
Can we rename this Visualize since there is no such thing as "KibanaApp" from user's pov?

Reply:
@rayafratkina We already have a section called "Visualizations" should these categories be combined?  If we want to change the name, I should update how the github_report script prints PRs with the KibanaApp label.